app/controllers/api/v1/groups_controller.rb

```diff
@@ -1,31 +1,31 @@
-class GroupController < ApplicationController
-  before_action :set_group, only: [:show, :update, :destroy]
+class Api::V1::GroupsController < ApplicationController
+  before_action :set_group, only: [:get_group_by_id, :update_group, :remove_group]
 
   # GET /groups
-  def list
+  def get_groups
     @groups = Group.all
```

Comment:
I understand it is auto generated code, but the use of variable `@groups` and `@group` worries me. Is it thread safe?